pytket/extensions/qiskit/backends/aer.py

```diff
@@ -96,6 +96,10 @@ def _tket_gate_set_from_qiskit_backend(
         gate_set.add(OpType.Unitary1qBox)
         gate_set.add(OpType.Unitary2qBox)
         gate_set.add(OpType.Unitary3qBox)
+
+    gate_set.add(OpType.Reset)
+    gate_set.add(OpType.Measure)
+    gate_set.add(OpType.Conditional)
```

Comment:
I think not all qiskit simulators support these.

Reply:
Annoyingly non-unitary ops such as measures and resets don't appear in the `basis_gates` field for the qiskit backend. I'll find some other way to check this.

Which simulators don't support these ops?

Reply:
Given that we only support three wrapped aer simulators from pytket-qiskit (Unitary, statevector and standard shots based) I opted to just not add the additonal ops to the unitary simulator and added them to the other two.